homeassistant/components/hue/config_flow.py

```diff
@@ -3,11 +3,12 @@
 import json
 import os
 
-from aiohue.discovery import discover_nupnp
 import async_timeout
 import voluptuous as vol
+from aiohue.discovery import discover_nupnp
 
 from homeassistant import config_entries
+from homeassistant.components.ssdp import ATTR_MANUFACTURERURL
```

Comment:
This shouldn't be moved. It caused some issue, I don't remember what right now.

Reply:
How can #27284 be completed then ?

Reply:
An adjustment has been made in #27362 to allow for these cases. Please rebase this PR.